### ISSUE TITLE
Make a broomstick hudbar

### ISSUE THREAD

**Issue by ObaniGemini** [MinetestForFun/server-minetestforfun#523]
It's highly possible that we could make broomstick's timer a hudbar.
I think we could even make so the hudbar IS the timer, then if the hudbar gets to 0, it is deleted/hidden and the effect gets to its end. If an icon is needed, I think the one for the broomstick would be self-explicit.

I thought to it seeing [this video](https://youtu.be/tcftWgKowrs?t=12m9s)

**Comment by Darcidride** [comment]
Good idea, but in this case we should delete the chat messages which was here for this reason

In fact, a small timer hud bar with the amount of seconds could be interesting and more readable for players
